pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -150,7 +155,24 @@ func (r *ReconcileChaosEngine) Reconcile(request reconcile.Request) (reconcile.R
 		}
 		return reconcile.Result{}, err
 	}
+	deletetimeStamp := engine.Instance.ObjectMeta.GetDeletionTimestamp()
 
+	if engine.Instance.Spec.EngineStatus == "stop" ||
+		engine.Instance.Spec.EngineStatus == "stopped" ||
+		deletetimeStamp != nil {
+		return r.reconcileForDelete(request)
+	} else if engine.Instance.Spec.EngineStatus == "start" {
+		r.recorder.Eventf(engine.Instance, corev1.EventTypeNormal, "Started reconcile for chaosEngine", "Will create all Chaos resources for engineUID: %v", string(engine.Instance.UID))
+		err := r.addFinalzerToEngine(engine, "chaosengine.litmuschaos.io/finalizer")
+		if err != nil {
+			reqLogger.Error(err, "Unable to add Finalizers in ChaosEngine Resource")
+		}
+		// Patch Status to init
+		err = r.updateStatus(engine, "initilized")
+		if err != nil {
+			reqLogger.Error(err, "Unable to Update Status in ChaosEngine Resource")
```

Comment:
Update the log message with this
Failed while updating chaosengine <engine-name> with error: err

Reply:
Adding error string, have used this `Unable to ....` approach in the executor, might be good to keep it that way.

Reply:
okay!! then it's fine